### PR TITLE
Sapi testing

### PR DESCRIPTION
**Pull request by sapikara**
Andy can now check the unit tests of test_cmf_faults.py.

### CODE REVIEW

File: src/eq_fault_geom/geomio/test_cfm_faults.py [hunk 1]
```diff
@@ -234,7 +235,7 @@ def test_dip_dir_str(self):
 
         dip_dir = None
         self.cmf_fault.dip_dir_str = dip_dir
-        self.assertEqual(self.cmf_fault.dip_dir, 330.15406806735484)
```
Comment (andy22b) [inline src/eq_fault_geom/geomio/test_cfm_faults.py]: You should use assertAlmostEqual here. The old value works fine on my machine.

File: src/eq_fault_geom/geomio/test_cfm_faults.py [hunk 2]
```diff
@@ -266,7 +267,7 @@ def test_validate_dip_direction(self):
         self.cmf_fault.dip_dir_str = dip_dir
 
         self.cmf_fault.validate_dip_direction()
-        self.assertEqual(self.cmf_fault.dip_dir, 150.1540680673612)
```
Comment (andy22b) [inline src/eq_fault_geom/geomio/test_cfm_faults.py]: assertAlmostEqual

**Review by andy22b — CHANGES_REQUESTED** [review]
We need to use assertAlmostEqual to address differences associated with different machines.

File: src/eq_fault_geom/geomio/test_cfm_faults.py (hunk 2 above)
Comment (sapikara) [reply src/eq_fault_geom/geomio/test_cfm_faults.py]: done - Thanks

**Pull request closed without merging**